tests/test_resources/test_clusters/test_on_demand_cluster.py

```diff
@@ -274,6 +274,37 @@ def test_status_cluster_rh_daemon_stopped(self, cluster):
             assert get_status_data["status"] == ResourceServerStatus.terminated
         cluster.restart_server()
 
+    @pytest.mark.level("minimal")
+    @pytest.mark.clustertest
+    def test_status_scheduler_basic_flow(self, cluster):
+
+        cluster_uri = rh.globals.rns_client.format_rns_address(cluster.rns_address)
+        headers = rh.globals.rns_client.request_headers()
+        api_server_url = rh.globals.rns_client.api_server_url
+
+        cluster.save()
+        # the scheduler start running in a delay of 1 min, so the cluster startup will finish properly.
+        # Therefore, the test needs to sleep for a while.
+        time.sleep(60)
```

Comment:
i think this same test is included in a different PR? also left a note about avoiding the sleep here

Reply:
yea correct, will revert the change here, and leave it in [1097](https://github.com/run-house/runhouse/pull/1097)